src/config/types.ts

```diff
@@ -125,6 +124,15 @@ export type Vote = CandidateVote | YesNoVote
 export type OptionalVote = Vote | undefined
 export type VotesDict = Dictionary<Vote>
 
+export interface WriteInCandidateTally {
+  name: string
+  tally: number
+}
+export type TallyCount = number
+export type CandidateVoteTally = (TallyCount | WriteInCandidateTally)[]
```

Comment:
Should write-ins be separate? Like, if this were an object:

```ts 
interface CandidateVoteTally {
  candidates: TallyCount[]
  writeIns: WriteInCandidateTally[]
}
```

It might help make some of the casting unnecessary and potentially less confusing.

Reply:
See https://github.com/votingworks/bmd/pull/791